plugins/modules/netbox_service.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2019, Kulakov Ilya  (@TawR1024)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.1",
+    "status": ["preview"],
+    "supported_by": "community",
+}
+
+DOCUMENTATION = r"""
+---
+module: netbox_service
+short_description: Creates or removes service from Netbox
+description:
+  - Creates or removes service from Netbox
+notes:
+  - This should be ran with connection C(local) and hosts C(localhost)
+author:
+  - Mikhail Yohman (@FragmentedPacket)
+  - Kulakov Ilya (@TawR1024)
+requirements:
+  - pynetbox
+version_added: '2.8'
```

Comment:
Change version added to 0.1.5 since that is what it will be released under for collections.